data/assets/scene/solarsystem/heliosphere/2012/sun_earth_2012_fieldlines_batsrus.asset

```diff
@@ -65,8 +65,9 @@ local BatsrusJ12OpenClosed = {
     Type = "RenderableFieldlinesSequence",
     SourceFolder = unzippedDataDestination.openClosed,
     InputFileType = "Osfls",
+    ShowAtAllTimes = false,
     Color = { 0.7, 0.4, 0.0, 0.6 }, -- Default color
-    ColorMethod = "By Quantity", -- Color by Quantity
+    ColorMethod = "ByQuantity", -- Color by Quantity
```

Comment:
Why remove the whitespace here? Leads to a breaking change, and we commonly use option properties with a space in the keys.

Maybe bring back the space?

Reply:
Good question. I believe I attempted to make it more explicit by introducing the enum Parameters::ColorMethod which I map to RenderableFieldlinesSequence::ColorMethod. An element in an enum cannot have spaces.
Currently on master it's a string and any value other than "Uniform" will make it use "ByQuantity".

Reply:
Great, then it will make it more explicit _and_ keep it like it was

Reply:
Though, I don't like the name. It feels pretty Swenglish. I think it should be By Property. It's a data property, like temperature or pressure.  I'll change it to this unless you have interventions